build_deploy.sh

```diff
@@ -99,4 +103,31 @@ make \
   internal_image_registry="quay.io" \
   image_repository="$QUAY_ORG/kas-fleet-manager" \
   docker/login \
-  image/push
\ No newline at end of file
+  image/push
+
+# create new relaase in managed-kafka-versions repo for kas-fleet-manager
+if [[ ! -z "$AUTHOR_EMAIL" ]] && [[ ! -z "$AUTHOR_NAME" ]] && [[ ! -z "$GITLAB_TOKEN" ]] && \
+[[ ! -z "$PASSWORD" ]] && [[ ! -z "$USERNAME" ]]; then
+  LATEST_COMMIT=$(git rev-parse HEAD)
+  echo "$LATEST_COMMIT"
+  SECOND_LAST_COMMIT=$(git rev-parse HEAD~1)
```

Comment:
Thanks @pawelpaszki 
I had missed out on the MR message. 

I think this needs improvement because:
1. We do not always do atomic commits and an MR might land that has tons of commits 
2. With the sync script, we always sync a batch of commits so HEAD~1 might not always guarantee that we have the latest.

I propose to base of this value from the `CURRENT_COMMIT_SHA` (computed from the version stream) or the `GIT_PREVIOUS_COMMIT` or `GIT_PREVIOUS_SUCCESSFUL_COMMIT` Jenkins variable if we have them.

Reply:
Perfect, thanks a lot.